### DISCUSSION TITLE
3章12節「HTTPサーバー」練習問題のコーディング

### DISCUSSION THREAD

**Discussion by satsukizzz** [satsukizzz/n-playspace-http#16] [Show and tell]
こちらもやっておこうと思います！コード書く方募集します 🙇

**Comment by Megafriday** [comment]
このコードの修正、担当してみたいです。
授業のやり方とは違い、フォークとかしないで、直接クローンしてからfeatureブランチ作成して作業を始めれば良いのでしょうか？
初めてなので、色々聞いてしまうかも知れませんがよろしくお願い致します。

**Reply by satsukizzz** [reply]
その通りです！
Collaboratorになってもらいましたので、その流れで作業できます。
(forkしてcloneしたリポジトリと混同させそうで恐縮ですが)
